### DISCUSSION TITLE
posts/btydyq/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#34] [Comments]
# posts/btydyq/

《被讨厌的勇气》是毕业后读完的第一本非技术类书籍，阅读时的原句摘录放在了学习博客中，本文是读后感。
[&hellip;] 《被讨厌的勇气》以一位青年与主张「世界极其简单，人们随时可以获得幸福」的哲人辩论的形式展开，通过哲人之口讲述阿德勒心理学。幸福感是一种综合感受，受制于短板效应。要获得幸福，必须要补齐短板。因此，本书覆盖了人际关系、个人成长、教育、人生意义等诸多话题。
[&hellip;] 书本 &hellip;

https://shitao5.org/posts/btydyq/

**Comment by hongtaoh** [comment]
很好的总结。这本书我只读了开头，还没看完。我总是学不会拒绝别人。

**Reply by Shitao5** [reply]
按照这本书的说法，提出请求是他人的课题，接受或拒绝是我们的课题。我们无法干涉他人的课题，但可以管理好自己的课题。按照我自己的理解，提出请求的人都是有被拒绝后的方案的，我们可以大胆拒绝。
